### PR TITLE
Feature/change form success behavior

### PR DESCRIPTION
**Pull request by codex-scribe**
Have implemented the 3 changes above.
For the following issue: https://github.com/fostyler/fos-flagship/issues/4

To test: 
1. open homepage
2. fill out the form including all required fields and click submit. 
3. Notice that the submit button will be disabled while the form is being submitted
4. if the submission is successful, the form will be cleared upon completion and a success message will display in a box, which disappears after 4 seconds.

@LailaAli @fostyler

### CODE REVIEW

File: src/components/ContactForm/index.js [hunk 1]
```diff
@@ -49,44 +59,57 @@ const ContactForm = () => {
...
       errorMessages.push('Message required!')
     }
+    //success logic
     if (errorMessages.length === 0) {
       const res = await serviceMethods.submitContactForm(formData)
       if (res.ok) {
         const { message } = await res.json()
         if (message === 'contactForm-create-success') {
           setIsFormSubmitted(true)
+          console.log(formRef.current)
```
Comment (fostyler) [inline src/components/ContactForm/index.js:69]: Please remove console logs

File: src/components/ContactForm/index.js [hunk 2]
```diff
@@ -49,44 +59,57 @@ const ContactForm = () => {
...
+      errors.length !== 0 &&
         <div className="notification" id="errorBox">
           <div className="columns is-vcentered is-mobile">
             <div className="column is-narrow">
               <img src="X-indicator.png" />
             </div>
             <div className="column">
               <p className="has-text-weight-bold errorBoxText headlineFont">Error</p>
-              {errors.map((error) => <p className="errorBoxText messageFont">{error}</p>)}
+              {errors.map((error, i) => <p className="errorBoxText messageFont" key={i}>{error}</p>)}
```
Comment (fostyler) [inline src/components/ContactForm/index.js:95]: You wouldn't want to use just a number for a key, add a relevant string to it

```
key={`errors-${i}`}
```

**Review by fostyler — CHANGES_REQUESTED** [review]
Hey @codex-scribe just a few clean-up tasks here.

File: src/components/ContactForm/index.js (hunk 1 above)
Comment (codex-scribe) [reply src/components/ContactForm/index.js:69]: Done.

File: src/components/ContactForm/index.js (hunk 2 above)
Comment (codex-scribe) [reply src/components/ContactForm/index.js:95]: Done.

**Review by fostyler — APPROVED** [review]

**Pull request merged**